apps/pie-docs/test/system/colour-toggle.spec.js

```diff
@@ -0,0 +1,37 @@
+import { test, expect } from '@playwright/test';
+import { disableCookieBanner } from '../playwright/playwright-helper';
+
+test.beforeEach(async ({ page, baseURL }) => {
+    await page.goto(baseURL);
+});
+
+test.describe('PIE - colour page toggle - @desktop', () => {
+    test.beforeEach(async ({ page, context }) => {
+        await disableCookieBanner(page, context);
+    });
+
+    test('Should check the functionality of the colour page toggle', async ({ page }) => {
+        // Arrange
+        await page.goto('foundations/colour/tokens/alias/light');
+
+        const toggleOptionA = page.getByTestId('toggle-option-a');
+        const toggleOptionB = page.getByTestId('toggle-option-b');
+
+        const lightTokens = page.locator('[id="toggle-content-a"]');
+        const darkTokens = page.locator('[id="toggle-content-b"]');
+
+        // Act - Clicks the dark theme toggle
+        await toggleOptionB.click();
+
+        // Assert - Dark tokens are visible and the light tokens are not visible
+        await expect.soft(darkTokens).toBeVisible();
```

Comment:
out of interest what does `soft` do?

Reply:
the `soft` makes it into a soft assertion. Meaning that it can assert on multiple things in a test that are relevant and won't break out of the testing until a normal assertion is met.

We do this to thoroughly test certain elements such as visibility in this instance.